celery/backends/database/__init__.py

```diff
@@ -98,11 +99,23 @@ def __init__(self, dburi=None, engine_options=None, url=None, **kwargs):
                 'Missing connection string! Do you have the'
                 ' database_url setting set to a real value?')
 
+        self.session_manager = SessionManager()
+
+        create_tables_at_setup = conf.get("database_create_tables_at_setup")
```

Comment:
Since `database_create_tables_at_setup` is a part of conf object, I don't think a lookup using `get` method is required anymore.
Thus the attribute can be accessed directly using `conf.database_create_tables_at_setup` .

Reply:
done :)